ethcore/src/ethereum/mod.rs

```diff
@@ -112,6 +112,11 @@ pub fn new_sokol<'a, T: Into<SpecParams<'a>>>(params: T) -> Spec {
 	load(params.into(), include_bytes!("../../res/ethereum/poasokol.json"))
 }
 
+/// Create a new Callisto chaun spec
+pub fn new_callisto<'a, T: Into<SpecParams<'a>>>(params: T) -> Spec {
```

Comment:
Maybe add a comment with a link to the contract pointing at the specific git hash that's generating the bytecode: https://github.com/EthereumCommonwealth/block-reward/blob/4cfc095830d267833c89797629010bfa1b5cbeac/contracts/CallistoBlockReward.sol ?

Reply:
nit: chain spec